tests/test_models/test_holt_winters_model.py

```diff
@@ -251,3 +252,23 @@ def test_components_sum_up_to_target(
     pred = model.predict(pred_df)
 
     np.testing.assert_allclose(np.sum(components.values, axis=1), pred)
+
+
+@pytest.mark.parametrize(
+    "model, expected_length",
+    [
+        (HoltWintersModel(), 3),
+        (HoltWintersModel(seasonal="add"), 4),
+        (HoltModel(), 2),
+        (SimpleExpSmoothingModel(), 0),
+    ],
+)
+def test_params_to_tune(model, expected_length):
```

Comment:
Should we try to fit the model with sampled parameters? Some of the errors pop out only at this stage.